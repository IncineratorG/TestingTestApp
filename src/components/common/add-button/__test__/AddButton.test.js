import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import AddButton from '../AddButton';

it('AddButton renders correctly', () => {
  const tree = renderer.create(<AddButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
