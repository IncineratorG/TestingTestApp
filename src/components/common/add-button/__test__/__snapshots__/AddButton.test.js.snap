// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "borderRadius": 38,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#4a9dec",
          "borderRadius": 38,
          "elevation": 8,
          "height": 75,
          "justifyContent": "center",
          "width": 75,
        },
        Object {
          "display": "flex",
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/icons/images/cross.png",
        }
      }
      style={
        Object {
          "transform": Array [
            Object {
              "rotate": "45deg",
            },
            Object {
              "scale": 0.3,
            },
          ],
        }
      }
    />
  </View>
</View>
`;
